### PR TITLE
Allow to remove keybinds + clean up keybinds + king keybinds added

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Clean up useless keybinds, added some code to allow dynamic removal of keybinds. This also allow to TM keybinds.

Also give king keybinds

## Changelog
:cl:
code: Clean up keybinds
qol: King get keybinds
fix: Spawn hugger keybind fixed
/:cl:

### CODE REVIEW

File: code/__DEFINES/dcs/signals.dm [hunk 1]
```diff
@@ -579,13 +571,12 @@
...
 #define COMSIG_XENOABILITY_UNRELENTING_FORCE "xenoability_unrelenting_force"
 #define COMSIG_XENOABILITY_UNRELENTING_FORCE_SELECT "xenoability_unrelenting_force_select"
-//#define COMSIG_XENOABILITY_NIGHTFALL "xenoability_nightfall"
 
 #define COMSIG_XENOABILITY_RAVAGER_CHARGE "xenoability_ravager_charge"
 #define COMSIG_XENOABILITY_RAVAGE "xenoability_ravage"
 #define COMSIG_XENOABILITY_RAVAGE_SELECT "xenoability_ravage_select"
 #define COMSIG_XENOABILITY_SECOND_WIND "xenoability_second_wind"
-#define COMSIG_XENOABILITY_IGNORE_PAIN "xenoability_ignore_pain"
+#define COMSIG_XENOABILITY_ENDURE "xenoability_ignore_pain"
```
Comment (BraveMole) [inline code/__DEFINES/dcs/signals.dm:579]:
```suggestion
#define COMSIG_XENOABILITY_ENDURE "xenoability_endure"
```

File: code/datums/keybinding/human.dm [hunk 2]
```diff
@@ -10,15 +10,6 @@
 	description = ""
 	keybind_signal = COMSIG_KB_QUICKEQUIP
 
-
-/datum/keybinding/human/holster
-	hotkey_keys = list("H")
-	name = "holster"
-	full_name = "Holster"
-	description = ""
-	keybind_signal = COMSIG_KB_HOLSTER
```
Comment (SplinterGP) [inline code/datums/keybinding/human.dm:19]: don't remove, it will be used again when someone fixes holsters

File: code/datums/keybinding/xeno.dm [hunk 3]
```diff
@@ -21,12 +21,6 @@
 	description = "Drop weeds to help grow your hive."
 	keybind_signal = COMSIG_XENOABILITY_DROP_WEEDS
 
-/datum/keybinding/xeno/choose_resin
-	name = "choose_resin"
-	full_name = "Choose Resin Structure"
-	description = "Selects which structure you will build with the (secrete resin) ability."
-	keybind_signal = COMSIG_XENOABILITY_CHOOSE_RESIN
```
Comment (SplinterGP) [inline code/datums/keybinding/xeno.dm:28]: ?

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/datums/keybinding/xeno.dm (hunk 3 above)
Comment (BraveMole) [reply code/datums/keybinding/xeno.dm:28]: this doesn't do anything anymore

File: code/datums/keybinding/human.dm (hunk 2 above)
Comment (BraveMole) [reply code/datums/keybinding/human.dm:19]: you want to keep a non-functionning keybind, that takes 10 seconds to remake? 

People will just keybind it and think it's a bug, which it is

Comment (SplinterGP) [reply code/datums/keybinding/human.dm:19]: keeps people from having to rebind it if they had something else bound.

File: code/modules/client/preferences_savefile.dm [hunk 4]
```diff
@@ -234,9 +234,9 @@
 	show_typing		= sanitize_integer(show_typing, FALSE, TRUE, initial(show_typing))
 	ghost_hud 		= sanitize_integer(ghost_hud, NONE, MAX_BITFLAG, initial(ghost_hud))
 	windowflashing	= sanitize_integer(windowflashing, FALSE, TRUE, initial(windowflashing))
-	auto_fit_viewport= sanitize_integer(auto_fit_viewport, FALSE, TRUE, initial(auto_fit_viewport))
-	widescreenpref = sanitize_integer(widescreenpref, FALSE, TRUE, initial(widescreenpref))
-	pixel_size = sanitize_float(pixel_size, PIXEL_SCALING_AUTO, PIXEL_SCALING_3X, 0.5, initial(pixel_size))
+	auto_fit_viewport = sanitize_integer(auto_fit_viewport, FALSE, TRUE, initial(auto_fit_viewport))
+	widescreenpref  = sanitize_integer(widescreenpref, FALSE, TRUE, initial(widescreenpref))
+	pixel_size      = sanitize_float(pixel_size, PIXEL_SCALING_AUTO, PIXEL_SCALING_3X, 0.5, initial(pixel_size))
```
Comment (spookydonut) [inline code/modules/client/preferences_savefile.dm:239]: we dont do aligning like this anymore

**Review by spookydonut — CHANGES_REQUESTED** [review]

File: code/modules/client/preferences_savefile.dm (hunk 4 above)
Comment (BraveMole) [reply code/modules/client/preferences_savefile.dm:239]: it's ugly :(

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**